sdcard/rootfs/kube-systemd/usr/lib/systemd/system/k8s-master.service

```diff
@@ -17,6 +17,7 @@ ExecStart=/bin/sh -c "exec docker run \
         -v /var/lib/docker/:/var/lib/docker:rw \
         -v /var/lib/kubelet:/var/lib/kubelet:shared \
         -v /var/run:/var/run:rw \
+        -v /var/log/containers:/var/log/containers:rw \
```

Comment:
In the title you're saying that the link should be `/var/lib/docker/containers/:/var/log/containers/`
What's correct?